geode-pulse/build.gradle

```diff
@@ -68,6 +68,11 @@ dependencies {
   testCompile project(':geode-core')
   testCompile files(project(':geode-core').sourceSets.test.output)
 
+  testCompile(group: 'com.codeborne', name: 'phantomjsdriver', version: '1.2.1') {
```

Comment:
define a phantomjsdriver.versionin dependency-versions.properties and refer that property here